lib/utilities.js

```diff
@@ -1,3 +1,5 @@
+const pluralizeImport = require('pluralize');
```

Comment:
`pluralizeLibrary` is also alright. but I get what you are doing here. was wondering why this name was changed, but seeing the exposed function makes sense.

Reply:
ya its a little ugly, totally open to other options, this was the best I could think of.